gitlab/integration.sh

```diff
@@ -130,10 +127,21 @@ set -e
 if [ $PIN_JUDGEDAEMON -eq 1 ]; then
     PINNING="-n 0"
 fi
+section_end more_setup
+
+section_start runguard_tests "Running isolated runguard tests"
```

Comment:
You're extending the test which takes the longest, can we move this to a dedicated job? Or is the extra time neglectable?

Reply:
It takes ~32s on CI, out of ~13mins. I would say it's not worth adding the complication of a dedicated job (until we can enable it on github actions).